packages/plugins/users-permissions/admin/src/pages/Providers/index.js

```diff
@@ -15,6 +15,9 @@ import {
 import has from 'lodash/has';
 import upperFirst from 'lodash/upperFirst';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
```

Comment:
Same here I would do it in the tests if there's no issue in the ui.

Reply:
ok @soupette  I can try into the test if it is possible

Reply:
done